l10n_it_ddt/models/stock_picking_package_preparation.py

```diff
@@ -353,6 +353,30 @@ def _prepare_invoice(self):
         })
         return res
 
+    @api.multi
+    def other_operations_on_ddt(self, invoice):
+        """ Once invoices are created with stockable products, we add them
+        all the invoiceable services available in the SO related to the
+        DDTs linked to the invoice.
+
+        Override this method in order to execute other additional operation on
+        the invoices created from DDT.
+        """
+        precision = self.env['decimal.precision'].precision_get('Account')
```

Comment:
```suggestion
        precision = self.env['decimal.precision'].precision_get('Product Unit of Measure')
```
La precisione per il campo `qty_to_invoice` è un'altra: https://github.com/odoo/odoo/blob/d3affc57988f3bde23f9ad5eaa008237b9acf765/addons/sale/models/sale.py#L783-L785